primitives/vector.py

```diff
@@ -1,5 +1,6 @@
-import numpy as np
+#!/usr/bin/env python3
 
+import numpy as np
 
```

Comment:
Another minor detail, technically there should be two blank lines between the imports and the class definition